### ISSUE TITLE
Improve error message when allow-picked-version = false

### ISSUE THREAD

**Issue by jugmac00** [buildout/buildout#481]
When you set "allow-picked-versions=false" you get a message like:
```
Error: Picked: pytest-instafail = 0.4.1
```
... which is not helping much.

Luckily, I found help in the plone docs, cf https://docs.plone.org/manage/troubleshooting/buildout.html#error-picked-some-package-some-version

```
This means that your buildout has “allow picked versions” set to false. You need to pin the version for the picked version (or turn on “allow picked versions”).
```

This was already reported 2012, but closed 2016 cf #35

**Comment by gotcha** [comment]
@jugmac00 Would you make a suggestion how to improve the error message ?

**Comment by mgedmin** [comment]
How about something like

```
Error: pytest-install does not have a version pin, and allow-picked-versions = false

To resolve this add
    pytest-install = 0.4.1
to [versions], or set allow-picked-versions = true
```
?

**Comment by gotcha** [comment]
Released in 3.0.0b2

**Comment by jugmac00** [comment]
I am looking forward to be able to use buildout 3 sooooo much :-)

```
STDERR
While:
  Updating pytest.
  Getting distribution for 'greenlet!=0.4.17'.
Error: Picked: greenlet = 1.0.0
```

What? 1.0.0 is not 0.4.1.7... ah, right :-)
